<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFImFCCSRelativeCCFDialog</class>
 <widget class="QDialog" name="QFImFCCSRelativeCCFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate Relative CCF Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0,0">
     <item row="2" column="1">
      <widget class="QFRDRComboBox" name="cmbCCF"/>
     </item>
     <item row="1" column="1">
      <widget class="QFRDRComboBox" name="cmbACF">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CCF file:</string>
       </property>
       <property name="buddy">
        <cstring>cmbCCF</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ACF file:</string>
       </property>
       <property name="buddy">
        <cstring>cmbACF</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>average channels:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinAvg">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnNextFileSameRole">
       <property name="text">
        <string>next file, same &amp;role</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnNextFile">
       <property name="text">
        <string>next &amp;file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnNextACF">
       <property name="text">
        <string>&amp;next ACF</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,1" columnstretch="1,1">
     <item row="1" column="1">
      <widget class="QFRDRImageToRunPreview" name="widOverviewCCF" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>CCF overview</string>
       </property>
       <property name="buddy">
        <cstring>widOverviewCCF</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
       <item row="1" column="0">
        <widget class="QPushButton" name="btnStoreData">
         <property name="text">
          <string>&amp;store result</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/file_add.png</normaloff>:/lib/file_add.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnStoreDataAll">
         <property name="text">
          <string>&amp;store all results</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/file_add.png</normaloff>:/lib/file_add.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnReplot">
         <property name="text">
          <string>&amp;replot</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnStoreDataAllSameRole">
         <property name="text">
          <string>&amp;store all results, same role</string>
         </property>
         <property name="icon">
          <iconset resource="../../lib/lib.qrc">
           <normaloff>:/lib/file_add.png</normaloff>:/lib/file_add.png</iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ACF overview</string>
       </property>
       <property name="buddy">
        <cstring>widOverviewACF</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QFPlotter" name="pltData" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>rel. CCF amplitude</string>
       </property>
       <property name="buddy">
        <cstring>pltData</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFRDRImageToRunPreview" name="widOverviewACF" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/help.png</normaloff>:/lib/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../lib/lib.qrc">
         <normaloff>:/lib/exit.png</normaloff>:/lib/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFRDRComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFRDRImageToRunPreview</class>
   <extends>QWidget</extends>
   <header>qfrdrimagetorunpreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbACF</tabstop>
  <tabstop>cmbCCF</tabstop>
  <tabstop>btnStoreData</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnReplot</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeCCFDialog</receiver>
   <slot>replotImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStoreData</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeCCFDialog</receiver>
   <slot>addResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFImFCCSRelativeCCFDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>replotImages()</slot>
  <slot>addResult()</slot>
 </slots>
</ui>
